frontend/src/app/main/ui/dashboard/projects.cljs

```diff
@@ -244,11 +244,11 @@
                          :on-menu-close on-menu-close
                          :on-import on-import}]
 
-       [:span.info (str file-count " files")]
+       [:span.info (str file-count " files,")]
```

Comment:
Please remove these changes, I've fixed them here -> https://github.com/penpot/penpot/pull/2298/commits/55b23da25febfb93da026258b3e6a22b5bda1e88

Reply:
I understand your point nut this is a i18n issue that I solved in the commit I linked so please, let that part of this MR :)

Reply:
Oh, I couldn't able to understand that `i18n` thing. Sorry for that.